components/src/components/form/form.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import Form from './form';
+
+describe('Form component', () => {
+  it('render component', () => {
+    const mockFn = vi.fn();
+    render(<Form addCard={mockFn} />);
+
+    expect(screen.getByRole('textbox')).toBeInTheDocument();
+    expect(screen.getByRole('combobox')).toBeInTheDocument();
+  });
+
+  it('all fields are empty', () => {
+    const mockFn = vi.fn();
+    render(<Form addCard={mockFn} />);
+
+    fireEvent.click(screen.getByRole('button', { name: /Submit/i }));
+
+    expect(screen.getByText(/name cannot be blank/i)).toBeInTheDocument();
+    expect(screen.getByText(/no date chosen/i)).toBeInTheDocument();
+    expect(screen.getByText(/no artist chosen/i)).toBeInTheDocument();
+    expect(screen.getByText(/no genre/i)).toBeInTheDocument();
+    expect(screen.getByText(/no file chosen/i)).toBeInTheDocument();
+    expect(screen.getByText(/Information must be confirm/i)).toBeInTheDocument();
+  });
+
+  it('reset form', () => {
+    const mockFn = vi.fn();
+    render(<Form addCard={mockFn} />);
+
+    fireEvent.change(screen.getByRole('textbox'), { target: { value: 'Test' } });
+    fireEvent.click(screen.getAllByRole('radio')[1]);
+
+    fireEvent.click(screen.getByRole('button', { name: /reset/i }));
+
+    expect(screen.getByRole('textbox')).not.toHaveAttribute('value');
+    expect(screen.getAllByRole('radio')[1]).not.toHaveAttribute('checked');
+  });
+
+  it('invalid date', () => {
```

Comment:
Fix naming for tests
Tests should do something like

Should check invalid date when ......